README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (optional, required only for LLVM build; add to PATH)
 * [Python](https://www.python.org/downloads/) (optional, required only for LLVM build; add to PATH)
+* [Git] (https://git-scm.com/downloads) (Select "use Git from the Windows Command Prompt")
```

Comment:
@tambry It's not necessary ?
Of course if it is mandatory, if you can not install git you do anything at all, must be started with the most important thing is that the code has recovered, just logic.
is very clear no : To initialize the repository don't forget to execute git submodule update --init to pull the wxWidgets source.